static/app/components/assigneeSelector.tsx

```diff
@@ -296,17 +296,41 @@ class AssigneeSelector extends React.Component<Props, State> {
   renderNewDropdownItems(): ItemsBeforeFilter {
     const teams = this.renderNewTeamNodes();
     const members = this.renderNewMemberNodes();
+    const suggestedAssignees = this.renderSuggestedAssigneeNodes() ?? [];
+    const assigneeIds = new Set();
+    const filteredTeams: ItemsBeforeFilter = [];
+    const filteredMembers: ItemsBeforeFilter = [];
+
+    // filter out duplicates of Team/Member if also a Suggested Assignee
+    if (suggestedAssignees.length) {
+      suggestedAssignees.filter(assignee =>
+        assigneeIds.add(`${assignee.value.type}:${assignee.value.assignee.id}`)
+      );
+      teams.filter(team => {
```

Comment:
typically `.filter` is used to apply some criteria to a collection and it returns a new collection in which all items have passed that criteria. In this case you use filter to iterate thru an array, but it is more conventional to do something like

```suggestion
      const filteredTeams = teams.filter(team => assigneeIds.has(`${team.value.type}:${team.value.assignee.id})
```

the compiler might complain that you are reassigning a const variable so you can either make it not const or remove the if statement `if (suggestedAssignees.length)` as even if suggestedAssignees is empty, the code should still run fine, just on an empty set/array